### ISSUE TITLE
"Press return" when aborting a command with Ctrl-C

### ISSUE THREAD

**Issue by die-github-die** [vifm/vifm#1000]
vifm 0.13

When I run `:!tail -f %c` and then press Ctrl-C, it says "Press return ..." and waits for it. In 0.12, you could see the panes right away once you pressed Ctrl-C. I don't think I want to use *two* hotkeys just to terminate a program.

**Comment by xaizek** [comment]
I don't think anything changed in this regard between v0.12 and v0.13.  Note the following:
 * `:!!command` (two `!`) always tries to pause after running a command
 * `:!command` (one `!`) pauses after a command that returns a non-zero exit code which usually happens after printing an error message and it's useful to know things didn't work out well

Check that `tail` doesn't exit with an error code, e.g. by appending `|| echo $?  > ~/tail-exit-code`.  Adding `|| true` should get rid of `Press return ...` in that case, but maybe there is something related to `tail` that you want to address (it must be failing for some reason).

**Comment by die-github-die** [comment]
After downgrading to 0.12, I get the expected behavior: I don't have to press Enter; Ctrl-C is enough.

**Comment by die-github-die** [comment]
To reproduce:

`:!touch file`

`:!tail -f file`

Then press Ctrl-C. At this point, v0.12 and v0.13 behave differently.

**Comment by xaizek** [comment]
Right, I forgot to add `%n` and use of terminal multiplexer is relevant here.  The behaviour became more correct in v0.12.1 with 5f99019731ff04a998351eaf5d6d14319b3699bc change which I didn't bother to document.  The effect is that non-zero return code returned by a process and produced when it dies from a signal are treated uniformly now.  Adding `|| true` is the workaround as there is no option for controlling this.

**Comment by die-github-die** [comment]
Ok, I've applied the patch in reverse, and it's working fine now. I don't care for uniformness in the slightest, so I'm golden.
